src/WebJobs.Script/Description/FunctionInvokerBase.cs

```diff
@@ -208,7 +203,7 @@ protected virtual void Dispose(bool disposing)
                 {
                     _fileChangeSubscription?.Dispose();
 
-                    (LogInfo.TraceWriter as IDisposable)?.Dispose();
```

Comment:
Now delegate disposal to the logger itself, rather than reach into it and only partially dispose